jena-jdbc/jena-jdbc-driver-remote/src/test/java/org/apache/jena/jdbc/remote/results/TestRemoteEndpointResults.java

```diff
@@ -46,8 +47,8 @@
      */
     @BeforeClass
     public static void setup() throws SQLException {
-    	ServerTest.allocServer();
-        
+        ServerTest.allocServer();
+    	    HttpOp.setDefaultHttpClient(null);
```

Comment:
Hopefully, when a pooling client, not  caching one is used this is unnecessary. (AbstractRemoteEndpointResultSetTests)